### PR TITLE
Python SDK rename state context

### PR DESCRIPTION
**Pull request by RyanLassigBanks**

### CODE REVIEW

File: sdk/python/sawtooth_sdk/processor/context.py [hunk 1]
```diff
@@ -112,3 +112,7 @@ def delete(self, addresses, timeout=None):
             raise InvalidTransaction(
                 'Tried to delete unauthorized address: {}'.format(addresses))
         return response.addresses
+
+    get = get_state
```
Comment (boydjohnson) [inline sdk/python/sawtooth_sdk/processor/context.py]: Were these notes?

Comment (boydjohnson) [reply sdk/python/sawtooth_sdk/processor/context.py]: Nevermind. Though this doesn't seem like the greatest idea since `set` is a python builtin.

Comment (peterschwarz) [reply sdk/python/sawtooth_sdk/processor/context.py]: Make sure that thesee deprecated methods are documented.

Comment (aludvik) [reply sdk/python/sawtooth_sdk/processor/context.py]: @boydjohnson it's not overwriting python's set builtin method any more than it was before when we did `def set()`. This is just a more efficient way to make `set_state` and `set` equivalent.

**Pull request merged**